linux_os/guide/services/sssd/sssd_memcache_timeout/bash/shared.sh

```diff
@@ -0,0 +1,21 @@
+# platform = multi_platform_rhel, multi_platform_fedora
+
+# Include source function library.
+. /usr/share/scap-security-guide/remediation_functions
+
+SSSD_CONF="/etc/sssd/sssd.conf"
+MEMCACHE_TIMEOUT_REGEX="[[:space:]]*\[nss]([^(\n)]*(\n)+)+?[[:space:]]*memcache_timeout"
+NSS_REGEX="[[:space:]]*\[nss]"
+TIMEOUT="86400"
```

Comment:
Replace `TIMEOUT` with `var_sssd_memcache_timeout` as it is now configurable in `var_sssd_memcache_timeout.var`